kiali-server/templates/deployment.yaml

```diff
@@ -156,14 +156,19 @@ spec:
         - name: {{ .name }}
           mountPath: "{{ .mount }}"
         {{- end }}
+        - name: "kiali-multi-cluster-secret"
+          mountPath: "/kiali-remote-cluster-secrets/kiali-multi-cluster-secret"
+          readOnly: true
```

Comment:
One reason I could see for having one secret _per_ remote kubeconfig is that it would make it simpler for automation to create a standalone secret rather than modifying an existing one. Would you be able to use projected volumes to do this? Maybe with `Values.deployment.custom_secrets` ?

Reply:
> one secret _per_ remote kubeconfig is that it would make it simpler for automation to create a standalone secret rather than modifying an existing one.

Maybe I misunderstand what you mean, but "one secret _per_ remote kubeconfig" is what [clustering.clusters[*].secret_name](https://kiali.io/docs/configuration/kialis.kiali.io/#.spec.clustering.clusters[*].secret_name) already does (and that feature still exists - I didn't touch that. So the user has the choice to have one secret per cluster or use that one secret for all of them).